include/aws/http/private/h2_stream.h

```diff
@@ -69,6 +69,12 @@ struct aws_h2_stream {
         int64_t window_size_self;
         struct aws_http_message *outgoing_message;
         bool received_main_headers;
+
+        bool content_length_received;
+        /* The received content length, if any */
+        uint64_t content_length;
+        /* The total length of payload of data frame received */
+        uint64_t received_data_length;
```

Comment:
make it clear that these variables are about the "incoming" message and not the "outgoing" message

maybe:
```
/* Set if incoming message has content-length header */
bool has_incoming_content_length;
uint64_t incoming_content_length;
/* The total length of payload of data frame received */
uint64_t incoming_data_length;
```